.github/workflows/test.yaml

```diff
@@ -375,3 +375,49 @@ jobs:
 
       - name: run notebook
         run: jupyter nbconvert --to notebook --execute examples/text_classification/Text_Classification_using_Tensorflow_and_🔭_Galileo.ipynb
+
+  send-slack-notification:
+    runs-on: ubuntu-latest
+    if: ${{ always() }}
+    needs:
+      [
+        mltc-pytorch,
+        mltc-tensorflow,
+        ner-hf-trainer,
+        ner-pytorch,
+        ner-spacy,
+        ner-tensorflow,
```

Comment:
@franz101 are these all the notebooks we have?

Reply:
@elboy3 @jonathangomesselman I remember there were Inference notebooks too. Should we include those in CI?

Reply:
we didn't end up including the inference notebook in this repo since inference is enterprise only and the notebooks are testing with cloud features. we need to think more on how we can accomplish inf notebooks